modules/ROOT/pages/platform/api/overview.adoc

```diff
@@ -38,6 +38,7 @@ In the event of `5xx` server error responses, you may consider retrying the requ
 The response may include a `Retry-After` header with a suggestion of a suitable minimum delay before attempting to retry.
 
 Rate limiting is set to 125 requests per minute.
+Starting 27 Jan 2025, rate limiting for the `/oauth/token` endpoint will be adjusted to 15 requests per hour.
```

Comment:
This is oddly specific and is not information that should be communicated via docs.